components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
```

Comment:
Next.js makes it so `import React from 'react';` isn't required, pretty cool. Also, `PropsWithChildren` covers `ReactNode` as you'll see below.

```suggestion
import { FC, PropsWithChildren } from 'react';
```